readthedocs/projects/tasks.py

```diff
@@ -116,6 +116,8 @@ def run(self, pk, version_pk=None, build_pk=None, record=True,
 
         This is fully wrapped in exception handling to account for a number of failure cases.
         """
+        unhandled_failure = False
```

Comment:
Its good to store blank string here.

Reply:
Why not `False`?

My idea is "if it's False, nothing happens. If it's the user message, and unhandled exception happens"

Reply:
The argument for string is probably that we're treating it as a string below, which will reduce confusion to our future selves :)

Reply:
I personally don't like to do `if '':` but I will change to that if you think it's better.